tests/graphql/test_graphql_client.py

```diff
@@ -0,0 +1,57 @@
+import mock
```

Comment:
there were already a file tests/test_graphql_client.py 
Can we add the test inside in this new file too ?